spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -39,6 +46,42 @@
             page_occurence: '',
           )
         end
+
+        context 'with american-style birthday data' do
+          let(:pii) { { dob: '01/01/1970' } }
+
+          it 'renders data' do
+            get :show
+            expect(assigns(:pii)[:birthdate]).to eq('January 01, 1970')
+          end
+        end
+
+        context 'with american-style birthday data with dashes' do
+          let(:pii) { { dob: '01-01-1970' } }
+
+          it 'renders data' do
+            get :show
+            expect(assigns(:pii)[:birthdate]).to eq('January 01, 1970')
+          end
+        end
```

Comment:
based on the fact that these test cases passed with no changes.... I don't think this is related to removing `american_date` gem in an obvious way

Reply:
I needed to use better dates for test cases, one did actually fail, resolving.